stylesheet.css

```diff
@@ -26,14 +22,18 @@
 	color: SkyBlue;
 }
 
-.action-sh {
-	color: greenyellow;
+.button:checked {
+  /* using this custom definition till Gnome provides a default
+   * https://gitlab.gnome.org/GNOME/gnome-shell/-/issues/4757 */
+  color: LightSlateGray;
```

Comment:
This landed upstream now. The default style for current HEAD is
```css
    color: #eeeeec;
    background-color: #2d2d2d;
    border-color: #191919;
    text-shadow: none;
    icon-shadow: none;
    box-shadow: none;
```

and

```css
    color: #2e3436;
    background-color: #efedec;
    border-color: #cdc7c2;
    text-shadow: none;
    icon-shadow: none;
    box-shadow: none; 
```

for GNOME Classic. (You can support the latter by providing a `classic.css` file)

Reply:
awesome